packages/perseus/src/widgets/numeric-input/utils.ts

```diff
@@ -35,37 +33,105 @@ export const generateExamples = (
     answerForms: readonly PerseusNumericInputAnswerForm[],
     strings: PerseusStrings,
 ): ReadonlyArray<string> => {
-    const forms =
-        answerForms?.length !== 0
-            ? answerForms
-            : Object.keys(NumericExampleStrings).map((name) => {
-                  return {
-                      name: name,
-                      simplify: "required",
-                  } as PerseusNumericInputAnswerForm;
-              });
+    // If the Content Creator has not specified any answer forms,
+    // we do not need to show any examples.
+    if (answerForms.length === 0) {
+        return [];
```

Comment:
I wanted to call out that I've technically changed this functionality. 

Previously we would return a string with all of the example strings that we just never displayed. This seemed like unnecessary logic/computations so I've updated it to simply exit early in such a case.